cmd/json.rb

```diff
@@ -142,6 +143,9 @@ def download_bottle(hash, tag)
     resource.version hash["pkg_version"]
     resource.downloader.resolved_basename = bottle_filename
 
+    # Remember that this formula name should be mapped to the newly downloaded bottle
+    Formulary.register_json_bottle hash["name"], resource.downloader.cached_location
```

Comment:
```suggestion
    ENV["HOMEBREW_BOTTLE_JSON"] = "1"
    Formulary.map_formula_name_to_local_bottle_path hash["name"], resource.downloader.cached_location
```

based on my latest version.